pkg/fwdport/fwdport.go

```diff
@@ -154,11 +153,8 @@ func (pfo *PortForwardOpts) BuildTheHostsParams() {
 	pfo.HostsParams = &HostsParams{}
 	localServiceName := pfo.Service
 	nsServiceName := pfo.Service + "." + pfo.Namespace
-	fullServiceName := fmt.Sprintf("%s.%s.svc.cluster.local", pfo.Service, pfo.Namespace)
+	fullServiceName := fmt.Sprintf("%s.%s.svc.cluster.%s", pfo.Service, pfo.Namespace, pfo.Context)
```

Comment:
Wonder if these might get too long, GKE for instance makes really silly long names for contexts by default :(

Reply:
In the multi-cluster case, you must use something cluster-specific and I think it's nice if you get the same name whether you connect to multiple clusters or not.  The hosts will still be accessible using the shortened names.

I guess maybe the code should log a warning and not change the hosts file for any name over 255 characters.

Or allow specifying a suffix for each context, like:

`kubefwd services -x "cluster1.long.domain.suffix:cluster1.local" -x "cluster2.long.context.name:cluster2.local"`

Or maybe the `-d` option would only apply to the last context provided:

`kubefwd services -x "long.context.name1" -d "suffix1" -x "long.context.name2" -d "suffix2"`

Reply:
My initial work-around attempt was something along those lines (splitting into multiple kubefwd commands and giving each one a -d). I really liked that approach because i could provide "just enough" naming rather than the whole fully qualified name. Probably something for a different feature request PR :D

Also, I completely spaced out and realized I could just rename the contexts with kubectl config rename-context, so that makes life super easy.!

Reply:
Good point - a decent workaround is to rename or alias the contexts instead of adding more command line options.